src/spec/spec_helper.rb

```diff
@@ -7,8 +9,20 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:each) do
+    access_token_resp = '{"access_token":"e72e16c7e42f292c6912e7710c838347ae178b4a",
+    "scope":"repo,gist", "token_type":"bearer"}'
+    # https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/
+    stub_request(:post, 'https://github.com/login/oauth/access_token')
+      .to_return(body: access_token_resp)
+
+    stub_request(:get, 'https://api.github.com/user')
+      .to_return(body: '{"id": 1, "name": "doctor manhattan", "login": "drmanhattan"}')
+  end
+
```

Comment:
This should be in githubs_controller_spec.rb and it's a setup for one test, having it here will run this before each test in all test files, which isn't needed